install-environment-packages.sh

```diff
@@ -6,14 +6,14 @@ export PIPENV_VERBOSITY=-1
 # Run pipenv virtual environment
```

Comment:
Old comments. Do we even need `PIPENV_VERBOSITY` variable?

Reply:
Fixed all old comments and delete unused pipenv variables